essays.html

```diff
@@ -3,13 +3,10 @@
 <head>
     <meta charset="utf-8" >
     <title>CSC630: The Open Source Movement</title>
-<<<<<<< HEAD
     <link rel="stylesheet" type="text/css" href="./assets/styles/main.css" />
-=======
     <link rel="stylesheet" type="text/css" href='./assets/styles/main.css' />
->>>>>>> 7012f5eb49957b8b13f334495cffcb1f531f2ed6
```

Comment:
Might want to rebase; this has been fixed iirc and may cause _another_ merge conflict.